### ISSUE TITLE
config object was deprecated in v3

### ISSUE THREAD

**Issue by Janaka-Steph** [martinkasa/capacitor-secure-storage-plugin#38]
I still get the warning `Capacitor WebPlugin "SecureStoragePlugin" config object was deprecated in v3 and will be removed in v4.`, even though I use version "^0.6.2" with Capacitor v3

**Comment by pablomaurer** [comment]
because 

```
  constructor() {
    super({
      name: 'BarcodeScanner',
      platforms: ['web'],
    });
  }
```
needs to be removed.

**Comment by martinkasa** [comment]
fixed in v0.7.0
